configure.ac

```diff
@@ -87,17 +87,21 @@ GNOME_ABRT_VERSION=`cat ./gnome-abrt-version`
 AC_SUBST(PYTHON_CFLAGS)
 AC_SUBST(GNOME_ABRT_VERSION)
 
-PKG_CHECK_MODULES([GTK], [gtk+-3.0])
+PKG_CHECK_MODULES([GTK], [gtk+-3.0 >= 3.10.0])
 PKG_CHECK_MODULES([LIBREPORT_GTK], [libreport-gtk > 2.0.19-3])
 PKG_CHECK_MODULES([ABRT_GUI], [abrt_gui > 2.1.6])
-PKG_CHECK_MODULES([PYGOBJECT3], [pygobject-3.0])
+PKG_CHECK_MODULES([PYGOBJECT3], [pygobject-3.0 >= 3.29.1])
```

Comment:
Does RHEL even have this version? You might want to make sure this version of ABRT doesn't end up in rhel 7.x. For RHEL 8 (now that the beta is out and we can talk publicly) we use 3.28, so that's problematic if this wants to be in the point release.

Reply:
3.22 in 7.x, looks like it.

Reply:
you mean abrt 3.22 or pygobject? In any case, I guess double check with the rest of the abrt team, just wanted to make sure you were aware :)

Reply:
I indeed meant PyGObject.

Reply:
> […] You might want to make sure this version of ABRT doesn't end up in rhel 7.x.

AFAIK RHEL 7.x has gnome-abrt something like 0.7.* which is older than anything I've ever contributed to. I'd love it to be upgraded because it looks ugly and (guessing from automatic bug reports) it occasionally crashes but so far I thought that upgrades in RHEL are not possible and/or not wanted.

Well, I'm not sure about it, I have CentOS on ore or two machines and that's how I guess what's the current version in RHEL.

> For RHEL 8 (now that the beta is out and we can talk publicly) we use 3.28, so that's problematic if this wants to be in the point release.

I'm really curious what is inside RHEL 8 beta but it's an internal job of Red Hat to decide which version of gnome-abrt finally lands there.

Reply:
Oh indeed, for some reason I expected abrt to be mostly Red Hat...